bin/goldrush

```diff
@@ -194,7 +197,15 @@ endif
 	@echo "Clean Done"
 
 # Set-up pipelines
-run: path-tigmint-ntLink check-G check-reads clean
+run: 
+	mkdir -p $(prefix)
+	cd $(prefix) && ln -sf ../$(reads).f* $(reads).fastq && goldrush run-in-dir reads=$(reads) G=$(G) t=$(t) z=$(z) track_time=$(track_time) k=$(k) w=$(w) tile=$(tile) b=$(b) u=$(u) a=$(a) o=$(o) x=$(x) h=$(h) s=$(s) m=$(m) M=$(M) r=$(r) P=$(P) d=$(d) span=$(span) dist=$(dist) k_ntLink=$(k_ntLink) w_ntLink=$(w_ntLink) rounds=$(rounds) polisher=$(polisher) polisher_mapper=$(polisher_mapper) shared_mem=$(shared_mem)
+	ln -sf $(prefix)/$(p2).fa $(p2).fa
+	ln -sf $(prefix)/$(p2).$(polished_infix).fa $(p2).$(polished_infix).fa
+	ln -sf $(prefix)/$(p2).$(polished_infix).span$(span).dist$(dist).tigmint.fa $(p2).$(polished_infix).span$(span).dist$(dist).tigmint.fa
+	ln -sf $(prefix)/$(p2).$(polished_infix).span$(span).dist$(dist).tigmint.fa.k$(k_ntLink).w$(w_ntLink).ntLink-$(rounds)rounds.fa $(p2).$(polished_infix).span$(span).dist$(dist).tigmint.fa.k$(k_ntLink).w$(w_ntLink).ntLink-$(rounds)rounds.fa
+
```

Comment:
What do you think about having some sort of final message mentioning that the files were soft-linked and/or where the final assembly can be found? Just thinking now with these commands the existing message is hidden a bit more in the log?

Reply:
I have added two echo messages. Please tell me what you think.

Reply:
Looks good! I might just flip them, so the message about the final file is very last? (just thinking that's the most important one probably)